oclc.py

```diff
@@ -1290,14 +1292,26 @@ def process_item(self, identifier):
             self.log.info("Processing identifier %r", identifier)
 
             for metadata in self.api.info_for(identifier):
-                oclc_editions = metadata.primary_identifier.primarily_identifies
```

Comment:
Because I got rid of the earlier hack, I can no longer refer to `metadata.primary_identifier` as an `Identifier`.